src/main/java/com/verisign/storm/metrics/GraphiteMetricsConsumer.java

```diff
@@ -202,6 +237,33 @@ else if (dataPoint.value instanceof Number) {
     graphiteDisconnect();
   }
 
+  private Double convertToDoubleValue(String key, Object value) {
+    if (value != null) {
+      Double result;
+
+      if (value instanceof String) {
+        try {
+          result = Double.parseDouble((String) value);
+        }
+        catch (NumberFormatException e) {
+          LOG.warn("Unparseable metric with key {} and value of {} found. Discarding metric.", key, value);
```

Comment:
this message could be more clear and say specifically that it is trying to parse it into a Double